iroh-net/src/endpoint.rs

```diff
@@ -291,21 +280,17 @@ struct StaticConfig {
     secret_key: SecretKey,
     transport_config: Arc<quinn::TransportConfig>,
     keylog: bool,
-    concurrent_connections: Option<u32>,
 }
 
 impl StaticConfig {
     /// Create a [`quinn::ServerConfig`] with the specified ALPN protocols.
     fn create_server_config(&self, alpn_protocols: Vec<Vec<u8>>) -> Result<quinn::ServerConfig> {
-        let mut server_config = make_server_config(
+        let server_config = make_server_config(
```

Comment:
`create_server_config` and `make_server_config` are now practically equivalent, but I want to keep `make_server_config`, so it doesn't change the public API, and I want to keep `StaticConfig::create_server_config` because it's the better internal API.

Reply:
fwiw this `make_server_config` is used by the `Node` builder as well to make the RPC endpoint.  I'm not sure you can completely remove it.

Reply:
oh, that is actually the reason this is pub!  because it's another crate :(